protocol/Protocol.py

```diff
@@ -934,6 +952,14 @@ def in_LOGIN(self, client, username, password='', cpu='0', local_ip='', sentence
 
 				if (len(unsupported) > 0):
 					self.out_SERVERMSG(client, 'Unsupported/unknown compatibility flag(s) in LOGIN: %s' % (unsupported), True)
+					
+				# record flag stats
+				self.flag_stats['n_samples'] += 1
+				for flag in flags:
+					if flag in self.flag_stats:
+						self.flag_stats[flag] += 1
+					else:
+						self.flag_stats[flag] = 1					
 			try:
```

Comment:
flag_stats seems to be used nowhere

Reply:
Yes, see https://github.com/spring/uberserver/issues/270#issuecomment-432333604

Afaics the stat collection that already existed (for cmd counts) didn't actually output to anywhere. Without seeing inside the live server I don't know what the best place/way for it to output stats is. I could add a server cmd to just print in plain-text, what's your preference?

Reply:
its outputed to the log file on reload...

Reply:
Ah, missed that - I've added it for flag_stats too. Having to call a RELOAD just to print out stats seems a little strange to me.